### PR TITLE
JDG-5557 - Fixing queryCache example

### PR DESCRIPTION
**Pull request by fabriziosta**
ECMAScript 6 introduced the "let" statement, which should become the final specification. 

Also, the PLAIN auth mechanism is not supported any more by Infinispan, is it?

https://issues.redhat.com/browse/JDG-5557

### CODE REVIEW

File: documentation/asciidoc/topics/code_examples/queries.js [hunk 1]
```diff
@@ -1,9 +1,9 @@
-var infinispan = require('infinispan');
-var protobuf = require('protobufjs');
+const infinispan = require('infinispan');
```
Comment (rigazilla) [inline documentation/asciidoc/topics/code_examples/queries.js:1]: I'm not sure about the usage of `const`. I mean: almost all the vars in this example are just assigned once, so for uniformity I would say that they should all be declared as `const`, but this way the result would be pretty weird to me. What do you think?

Comment (fabriziosta) [reply documentation/asciidoc/topics/code_examples/queries.js:1]: You are right, the only variables that cannot be declared const are `protoMetaClient` and `client`, which are assigned within the try statement and used outside it.
I tried the example using const assignment where was possible to use it, and it's working great

**Comment by rigazilla** [comment]
@fabriziosta I've added a commit with a fix and a code improvement: moved clients vars at function level so they can be declared where they are used). If it's ok to you I can merge this

**Comment by fabriziosta** [comment]
@rigazilla it's perfect, thank you very much!

**Comment by rigazilla** [comment]
merged, thanks @fabriziosta !

**Pull request merged**